src/main/twirl/uk/gov/hmrc/sca/views/AttorneyBanner.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.ptafrontend.views.html._
+@import uk.gov.hmrc.ptafrontend.viewmodels.attorneybanner.AttorneyBanner
+@import uk.gov.hmrc.auth.core.retrieve.v2.TrustedHelper
+
+@this(
+  ptaAttorneyBanner: PtaAttorneyBanner
+)
+
+@(helper: TrustedHelper)(implicit messages: Messages)
+
+  @ptaAttorneyBanner(AttorneyBanner(name = helper.principalName, classes = "govuk-!-margin-top-2", accountUrl = helper.returnLinkUrl))
```

Comment:
Is there a need to create our own AttorneyBanner.scala if all we're doing is passing directly to the ptaAttorneyBanner? Maybe we can just inject PtaAttorneyBanner directly to ScaLayout, then in a future update we remove the play-frontend-pta dependency and make our own object

Reply:
removed the file